librz/core/cmd_descs/cmd_analysis.yaml

```diff
@@ -394,6 +394,68 @@ commands:
           - RZ_OUTPUT_MODE_RIZIN
           - RZ_OUTPUT_MODE_JSON
         args: []
+      - name: avg
+        summary: Global variables
+        subcommands:
+          - name: avg
+            summary: show global variables
+            type: RZ_CMD_DESC_TYPE_ARGV_STATE
+            cname: analysis_print_global_variable
+            modes:
+              - RZ_OUTPUT_MODE_STANDARD
+              - RZ_OUTPUT_MODE_JSON
+            args:
+              - name: var_name
+                type: RZ_CMD_ARG_TYPE_STRING
+                optional: true
+          - name: avga
+            summary: add global variable manually
+            type: RZ_CMD_DESC_TYPE_ARGV
+            cname: analysis_global_variable_add
+            args:
+              - name: var_name
+                type: RZ_CMD_ARG_TYPE_STRING
+              - name: addr
+                type: RZ_CMD_ARG_TYPE_RZNUM
+              - name: type
+                type: RZ_CMD_ARG_TYPE_STRING
```

Comment:
@XVilka shall we use `RZ_CMD_ARG_TYPE_ANY_TYPE` here?

Reply:
Yes, please.